exercises/elm-1-a/.meta/config.json

```
@@ -0,0 +1,21 @@
+{
+  "solution_files": ["src/AggregateScorers.elm"],
+  "tests": [
+    {
+      "name": "doesn't change a single scorer",
+      "cmd": "aggregateScorers [ \"Anna\" ]"
+    },
+    {
+      "name": "doesn't change multiple unique scorers",
+      "cmd": "aggregateScorers [ \"Anna\", \"Lisa\" ]"
+    },
+    {
+      "name": "converts duplicates in to goals",
```

I'm not entirely sure, but I think this should be:

```suggestion
      "name": "converts duplicates into goals",
```